### PR TITLE
Firefighting Additions

### PR DESCRIPTION
**Pull request by Cerebulon**
New firefighting equipment and sprites, some minor buffs to make them more viable as firefighting equipment. Including non-voidsuit stuff for planetside firefighting.

Extinguishers now contain firefighting foam and are refilled using extinguisher refill tanks or single-use refill canisters. Foam is more effective at putting out fires than water and will suppress fire on tiles where it is spread.

Foam remains on tiles for a brief time but is not slippery so firefighters will not own themselves constantly.
Now we just need to actually have fires ;)

:cl:
rscadd - New firefighting gear sprites.
rscadd - Added enhanced firefighting gear for atmospheric techs and chief engineer.
tweak - Fire extinguishers now use foam instead of water.
/:cl:

### CODE REVIEW

**Comment by Greenjoe12345** [comment]
is this foam still effective against slimes? or will xenobio get water filled ones to continue to use against em?

**Comment by Cerebulon** [comment]
> 
> 
> is this foam still effective against slimes? or will xenobio get water filled ones to continue to use against em?

Added the Xenoblaster, for blasting slimes. It's a super soaker.

File: code/game/objects/effects/decals/Cleanable/fuel.dm [hunk 1]
```diff
@@ -82,3 +82,51 @@
...
+			other.amount += src.amount
+			other.Spread()
+			has_spread = 1
+			break
+
+	if(!has_spread)
+		Spread()
+		QDEL_IN(src, 2 MINUTES)
+	else
+		qdel(src)
```
Comment (MistakeNot4892) [inline code/game/objects/effects/decals/Cleanable/fuel.dm:114]: `return INITIALIZE_HINT_QDEL`, pls donut qdel directly in Initialize().

**Comment by Spookerton** [comment]
no traffic in 7 months; author can reopen if conflict resolution etc is going to happen

**Pull request closed without merging**